### PR TITLE
cce fixes for network and data vols

### PR DESCRIPTION
**Pull request by tischrei**

### CODE REVIEW

File: tests/integration/targets/cce_cluster_node/tasks/main.yaml [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+          - node is success
+          - node is changed
+
+  always:
+    - block:
+      # Cleanup
+      - name: Drop CCE cluster node
+        opentelekomcloud.cloud.cce_cluster_node:
+          cluster: "{{ cce_cluster_name }}"
+          name: "{{ cce_node_name }}""
```
Comment (kucerakk) [inline tests/integration/targets/cce_cluster_node/tasks/main.yaml:128]: please remove double quote at the end

File: tests/integration/targets/cce_cluster_node/tasks/main.yaml [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+          state: "absent"
+      
+      - name: Drop keypair
+        openstack.cloud.keypair:
+          name: "{{ keypair_name }}""
```
Comment (kucerakk) [inline tests/integration/targets/cce_cluster_node/tasks/main.yaml:133]: please remove double quote at the end

File: tests/integration/targets/cce_cluster_node/tasks/main.yaml (hunk 1 above)
Comment (tischrei) [reply tests/integration/targets/cce_cluster_node/tasks/main.yaml]: done

File: tests/integration/targets/cce_cluster_node/tasks/main.yaml (hunk 2 above)
Comment (tischrei) [reply tests/integration/targets/cce_cluster_node/tasks/main.yaml]: done

**Review by lego963 — DISMISSED** [review]

**Review by kucerakk — APPROVED** [review]

**Review by lego963 — APPROVED** [review]

**Pull request merged**